flathub.json

```
@@ -0,0 +1,6 @@
+{
+    "only-arches": [
+        "x86_64"
+    ],
+    "publish-delay-hours": 0
```

```suggestion
    ]
```

This is what @barthalion was talking about.